localdev/seed/resources/seed_data.json

```diff
@@ -43,12 +49,15 @@
             "readable_id": "course-v1:xPRO+SysEngxB1",
             "course_runs": [
                 {
-                    "title": "Systems Engineering - Fall 2019",
+                    "title": "Architecture of Complex Systems - Fall 2019",
                     "courseware_id": "course-v1:TestX+SysEngxB1+1T2019",
```

Comment:
In production, we don't use `1T2019` style runs any more.  They are more of the form `R7`

Reply:
Is the "7" in "R7" meaningful in some way that I should capture in the new values?

Reply:
Leaving this alone for now as discussed